### ISSUE TITLE
unrecognized property 'documentation'

### ISSUE THREAD

**Issue by gianlucainnocentevegans** [env0/serverless-openapi-typescript#23]
Hello,
i have my serverless.ts with this function:
```
functions: {
		getPatients
 }
```

and the documentation section of the serverless.ts with this configuration:
```
documentation: {
      version: '1',
      title: 'Claire API',
      description: 'Claire API',
      securitySchemes: {},
      security: {},
      models: {}
    }
```

My getPatients function is imported and has this content:

```
import schema from './schema';
import { handlerPath } from '@libs/handler-resolver';

export default {
  handler: `${handlerPath(__dirname)}/handler.main`,
  events: [
    {
      http: {
        method: 'GET',
        path: 'patients',
        request: {
          schemas: {
            'application/json': schema,
          },
        },
      }
    }
  ],
};
```

The problem is i got this error: "Some functions have http events which are not documented: getPatients".

If i add this json to the http event:
```
documentation: {
          summary: "get patients",
          methodResponses: [
            {
              statusCode: 200,
              responseBody: "OK"
            }
          ]
        },
```

then i got this error: No root type "undefined.GetPatients.Response" found.
And this warning: Invalid configuration encountered
  at 'functions.getPatients.events.0.http': unrecognized property 'documentation'

Cacn anyone help me? Thank you

**Comment by djMax** [comment]
@gianlucainnocentevegans did you ever figure this out? Same thing is happening to me.
